lib/sufia/engine.rb

```diff
@@ -1,4 +1,12 @@
 module Sufia
+  class Statefile
+    def self.default
+      return '/tmp/minter-state' if Rails.env.development? || Rails.env.test?
+      raise NotImplementedError, "can't use the default statefile for production" unless Dir.exist?('/var/sufia')
```

Comment:
It shouldn't because if you'd specify that in config/initializers/sufia.rb, and it would skip this entirely. Of course, it wouldn't check to see if `/opt/sufia` was there, but that's not our problem, is it?